test/test.js

```diff
@@ -809,6 +809,16 @@ describe('Page', function() {
       expect(await waitForSelector).toBe(true);
       expect(divHidden).toBe(true);
     }));
+    it('hidden should wait for opacity: 0', SX(async function() {
```

Comment:
Tests have slightly changed; the `SX` function is no longer needed, and the `page` is passed to the test function itself.
```js
it('hidden should wait for opacity: 0', async function({page}) {
  // ...
});
```

Reply:
Thanks for the heads up.